pkg/tests/ossm/bug_istiopods_test.go

```diff
@@ -15,79 +15,49 @@
 package ossm
 
 import (
-	"fmt"
-	"math/rand"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/env"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-// TestIstioPodProbesFails tests that Istio pod get stuck with probes failure after restart. Jira ticket: https://issues.redhat.com/browse/OSSM-2434
+// TestIstioPodProbesFails tests that Istio pod get stuck with probes failure after restart. Jira ticket bug: https://issues.redhat.com/browse/OSSM-2434
 func TestIstioPodProbesFails(t *testing.T) {
```

Comment:
```suggestion
func TestIstiodPodProbesFails(t *testing.T) {
```

But the test name doesn't clearly say what it's testing. Sure it isn't verifying that probes fail, but the opposite.

Reply:
Change for TestIstioPodStuckAfterRestarts because the test is to delete multiple time the istiod pod to check that not get stuck starting